arelle/plugin/inlineXbrlDocumentSet.py

```diff
@@ -591,6 +633,16 @@ def commandLineOptionExtender(parser, *args, **kwargs):
                       dest="xbrliNamespacePrefix",
                       help=SUPPRESS_HELP,
                       type="string")
+    parser.add_option("--deduplicateIxbrlFacts",
+                      action="store",
+                      choices=[a.value for a in ValidateDuplicateFacts.DeduplicationType],
+                      dest="deduplicateIxbrlFacts",
+                      help=_("Remove duplicate facts when extracting XBRL instance."))
+    parser.add_option("--deduplicateixbrlfacts",  # for WEB SERVICE use
+                      action="store",
+                      choices=[a.value for a in ValidateDuplicateFacts.DeduplicationType],
+                      dest="deduplicateIxbrlFacts",
+                      help=SUPPRESS_HELP)
```

Comment:
This might be more cautious than necessary, but I think we should raise an exception [if another plugin implements and returns True](https://github.com/Arelle/EdgarRenderer/blob/3cf7f2e3e5eb621dd41b840f3244b0b16c21df7e/__init__.py#L1732) for the `InlineDocumentSet.SavesTargetInstance` hook. Those plugins could have unforeseen issues if combined with this parameter.

Reply:
Sounds reasonable. Let me know if my latest commit doesn't address your concern.